src/components/DifficultyExplanation/DifficultyExplanation.css

```diff
@@ -0,0 +1,19 @@
+.information{
+    display: flex;
+    position: absolute;
+    right: 25%;
+    width: 20%;
+    height: 30%;
```

Comment:
we don't need to have the height set to a certain value! since display is flex, then the box will grow to adjust to the size that the text inside will occupy